lib/smart_proxy_remote_execution_ssh.rb

```diff
@@ -21,6 +21,15 @@ def public_key_file
         File.expand_path("#{private_key_file}.pub")
       end
 
+      def cert_file
+        File.expand_path("#{private_key_file}-cert.pub")
+      end
```

Comment:
Should we require this strict naming convention for cert file
similarly as we do for public key file, or should we make this an editable
value in settings, with this format being the possible default?